src/mscorlib/src/System/AppDomain.cs

```diff
@@ -330,18 +330,12 @@ public String BaseDirectory
             }
         }
 
-        public override String ToString()
+        public override string ToString()
         {
-            StringBuilder sb = StringBuilderCache.Acquire();
-
-            String fn = nGetFriendlyName();
-            if (fn != null)
-            {
-                sb.Append(SR.Loader_Name + fn);
-                sb.Append(Environment.NewLine);
-            }
-
-            return StringBuilderCache.GetStringAndRelease(sb);
+            string fn = nGetFriendlyName();
+            return fn != null ?
+                SR.Loader_Name + fn + Environment.NewLine :
```

Comment:
Can we remove the trailing `Environment.NewLine`? Looks like the only reason it's there is from a [previous implementation](https://github.com/Microsoft/referencesource/blob/b31308b03e8bd5bf779fb80fda71f31eb959fe0b/mscorlib/system/appdomain.cs#L2210-L2233) where additional diagnostic information could be appended after it.